src/analytics/private/sqls/redshift/sp-upsert-users.sql

```diff
@@ -1,5 +1,5 @@
-CREATE OR REPLACE PROCEDURE {{schema}}.{{sp_upsert_users}}()
-AS 
+CREATE OR REPLACE PROCEDURE {{schema}}.{{sp_upsert_users}}() 
+NONATOMIC AS 
```

Comment:
please make sure it's safe for re-running the procedure again.

Reply:
yes.